README.md

```diff
@@ -1,10 +1,10 @@
-SecureXPC provides an easy way to perform secure XPC Mach service communication. 
-[`Codable`](https://developer.apple.com/documentation/swift/codable) conforming types are used to send messages and
-receive replies. This framework is ideal for communicating with helper tools installed via 
-[`SMJobBless`](https://developer.apple.com/documentation/servicemanagement/1431078-smjobbless).
+Use pure Swift to easily and securely communicate with XPC Services and XPC Mach services, with customized support for
+helper tools installed via [`SMJobBless`](https://developer.apple.com/documentation/servicemanagement/1431078-smjobbless).
+A client-server model is used with [`Codable`](https://developer.apple.com/documentation/swift/codable) conforming types
+to send messages to registered routes and receive replies.
 
-To see a runnable sample app using this framework, check out
-[SwiftAuthorizationSample](https://github.com/trilemma-dev/SwiftAuthorizationSample).
+macOS 10.10 and later is supported. Starting with macOS 10.15, clients can use `async` functions to make calls and
+servers can register `async` handlers for their routes.
```

Comment:
I would invert this to focus on the newer. It's novel, attention-grabbing and awesome.

> SecureXPC uses [Swift Concurrency](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html) to allow clients to make non-blocking, asynchronous calls to servers. A closure-based API is also available, for backwards compatibility for macOS versions prior to 10.15.